src/main/java/com/example/demo/Controllers/MovieController.java

```diff
@@ -1,24 +1,70 @@
 package com.example.demo.Controllers;
 
 import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.validation.Valid;
+
+import java.net.URI;
+import java.net.URISyntaxException;
 
 import com.example.demo.Models.Movie;
-import com.example.demo.Repositories.MovieRepository;
+import com.example.demo.Services.MovieService;
+
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.*;
+import org.springframework.hateoas.CollectionModel;
+import org.springframework.hateoas.EntityModel;
+import org.springframework.http.ResponseEntity;
+import org.springframework.hateoas.IanaLinkRelations;
+
 @RestController
 public class MovieController {
-    private final MovieRepository repository;
 
-    MovieController(MovieRepository repository) {
-        this.repository = repository;
-    }
+    @Autowired
+    private MovieService movieService;
 
     // Aggregate root
 
+    @GetMapping("/api/movies/{id}")
+    EntityModel<Movie> one(@PathVariable Long id) {
```

Comment:
You didn't add the access modifier for the methods, is that ok?
When we don't add the access modifier how Java interprets that?